keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -508,27 +508,64 @@ When a new `hostProcess` container is created with the name of a local users Gro
 
 #### Container Mounts
 
-- Window's bind-filter driver will be used to create a view that merges the host's OS filesystem with container-local volumes.
-When `hostProcess` containers are started a new volume will be created which contains the contents of the contaner image.
-This volume will be mounted to `c:\hpc`. The default working directory for `hostProcess` containers will also be set to `c:\hpc`.
-- Volume mounts (includinge service account tokens) will be supported for `hostProcess` containers and can be accessed just the same way as regular Windows Server containers.
-  - Named Pipe mounts will **not** be supported.
+There will be two different behaviors for how volume mounts are configured in `hostProcess` containers.
+
+- **Bind Mounts**
+  - With the approach Window's bind-filter driver will be used to create a view that merges the host's OS filesystem with container-local volumes.
+  - When `hostProcess` containers are started a new volume will be created which contains the contents of the contaner image.
+    This volume will be mounted to `c:\hpc`.
+  - Additional volume mounts specified for `hostProcess` containers will be mounted at their requested location and can be access the same way as volume mounts in Linx or regular Windows Server contaienrs.
```

Comment:
What is the behavior when the requested path conflicts with the host filesystem?

Reply:
updated the KEP